react-native/react/login/index.render.desktop.js

```diff
@@ -1,12 +1,69 @@
+/* @flow */
+/* eslint-disable react/prop-types */
+
 import React, {Component} from '../base-react'
+import {globalStyles} from '../styles/style-guide'
+import {resizeLoginForm} from '../local-debug'
+
+import {remote} from 'electron'
+
+import Carousel from '../util/carousel.desktop'
+
+import type {LoginRenderProps} from './index.render'
 
 export default class LoginRender extends Component {
-  render () {
+  props: LoginRenderProps;
+
+  window: ?any;
+  originalSize: ?{width: number, height: number};
+
+  componentWillMount () {
+    if (resizeLoginForm) {
```

Comment:
This is fine for now but i'm worried about a component doing this. Might be better for nav or something higher level to handle this?